libraries/matrix/impl/src/test/kotlin/io/element/android/libraries/matrix/impl/room/member/RoomMemberListFetcherTest.kt

```diff
@@ -204,32 +203,6 @@ class RoomMemberListFetcherTest {
     }
 }
 
-class FakeRustRoom(
-    private val getMembers: () -> RoomMembersIterator = { FakeRoomMembersIterator() },
-    private val getMembersNoSync: () -> RoomMembersIterator = { FakeRoomMembersIterator() },
-) : Room(NoPointer) {
-    var membersCallCount = 0
-    var membersNoSyncCallCount = 0
-
-    override fun id(): String {
-        return A_ROOM_ID.value
-    }
-
-    override suspend fun members(): RoomMembersIterator {
-        membersCallCount++
-        return getMembers()
-    }
-
-    override suspend fun membersNoSync(): RoomMembersIterator {
-        membersNoSyncCallCount++
-        return getMembersNoSync()
-    }
-
-    override fun close() {
-        // No-op
-    }
-}
-
 class FakeRoomMembersIterator(
```

Comment:
Maybe this should be moved too?

Reply:
Ah yes, probably

Reply:
Done in https://github.com/element-hq/element-x-android/pull/3482/commits/3d883916e6158f22a53eb35c6586b6dc42491b49